src/Symfony/Component/Console/Helper/ProgressBar.php

```diff
@@ -339,15 +339,23 @@ public function start($max = null)
      * Advances the progress output X steps.
      *
      * @param int $step Number of steps to advance
-     *
-     * @throws LogicException
      */
     public function advance($step = 1)
     {
         $this->setProgress($this->step + $step);
     }
 
     /**
+     * Regresses the progress output X steps.
+     *
+     * @param int $step Number of steps to regress
+     */
+    public function regress($step = 1)
```

Comment:
An exception would be a DC issue, forcing one to call the correct method with boilerplate just for the sign. I agree with the first comment, we need only progress.